Dev/GitHub/github-review-requests-TS-5m.ts

```diff
@@ -0,0 +1,231 @@
+#!/opt/homebrew/bin/deno run --allow-all
```

Comment:
@victorstein 

What do you think about making the shebang less M1 silicon Homebrew specific?

```suggestion
#!/usr/bin/env -S -P/${HOME}/.deno/bin:/opt/homebrew/bin:/usr/local/bin deno run --allow-all
```